hydradx/model/amm/centralized_market.py

```diff
@@ -83,7 +83,9 @@ def __init__(
             self,
             order_book: dict[tuple[str, str], OrderBook],
             asset_list: list[str] = None,
-            trade_fee: float = 0
+            trade_fee: float = 0,
+            preferred_stablecoin: str = 'USD',
```

Comment:
I'm using it in combine_swaps, but not in a major way. Basically if there are some assets left over to buy/sell after the first pass concludes, it will next try routing through that coin. I could just hardcode it instead